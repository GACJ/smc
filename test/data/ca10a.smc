Cambridge
x30x14x1250x36x1470x58x16x70x18x90 l12
5040 5040
144000
bob 8M 9W 0H
single 8M 9W 0H
parts 2
music hb10.mus
