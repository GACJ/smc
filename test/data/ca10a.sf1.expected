Cambridge
x30x14x1250x36x1470x58x16x70x18x90 l12
5040 5040
144000
bob pn14 8M 9W 0H
single pn1234 8M 9W 0H
part 2

MUSIC
3-4 coursing 
1000 0 134xxxxxxxc 143xxxxxxxc
3-4 in 3-4 position 
100 0 1x34xxxxxxc 1x43xxxxxxc
5-6 coursing 
1000 0 156xxxxxxxc 165xxxxxxxc
5-6 in 3-4 position 
1 0 1x56xxxxxxc 1x65xxxxxxc
5-6 in 4-5 position 
1 0 1xx56xxxxxc 1xx65xxxxxc
5-6 in 5-6 position 
1000 0 1xxx56xxxxc 1xxx65xxxxc
56 & 65s
10 0 xxxx567890 xxxx657890
23456s 
1000 0 xxxxx23456
65432s
1000 0 xxxxx65432
76543s
1000 0 xxxxx76543
12345s
1000 0 xxxxx12345
3456s
100 0 xxxxxx3456
6543s
100 0 xxxxxx6543
2345s
1 0 xxxxxx2345
5432s
1 0 xxxxxx5432
exclusions
-1 0 1xx34xxxxxc 1xx43xxxxxc 1xxx34xxxxc 1xxx43xxxxc

5040	161038	2	sH sWH sM sMH sW sMsWH sWsH 
5040	161038	2	sH sWH sMsW sMH sW sWH sWsH 
5040	167038	2	sH sMH sM sMH sW sWH sMsWsH 
5040	167038	2	sH sMH sM sMH sW sMsWH sMsH 
5040	167038	2	sH sMH sMsW sMH sW sWH sMsH 
5040	167038	2	sH sMH sMsW sMH sW sMsWH sMsWsH 
5040	161038	2	sH sMsWH sM sMH sW sWH sWsH 
5040	161038	2	sH sMsWH sMsW sMH sW sMsWH sWsH 
5040	155390	2	sWH M M MH W sMWsH WH 
5040	172876	2	sWH M MW W MsH sW MsWH 
5040	155390	2	sWH M MsW MH W WsH WH 
5040	160038	2	sWH sM sMH sW sWH sWsH sMsH 
5040	160038	2	sWH sM sMH sW sWH sMsW sMsW 
5040	160038	2	sWH sM sMH sW sMsWH sWsH sH 
5040	160038	2	sWH sM sMH sW sMsWH sM sMsW 
5040	160038	2	sWH sMsW sMH sW sWH sWsH sH 
5040	160038	2	sWH sMsW sMH sW sWH sM sMsW 
5040	160038	2	sWH sMsW sMH sW sMsWH sWsH sMsH 
5040	160038	2	sWH sMsW sMH sW sMsWH sMsW sMsW 
5040	156428	2	sWsH MH MsW sWsH M MsW MWsH 
5040	156428	2	sWsH MH MsW sMsWsH M M MWsH 
5040	167038	2	sWsH sMH sM sMH sW sWH sMsH 
5040	167038	2	sWsH sMH sM sMH sW sMsWH sMsWsH 
5040	167038	2	sWsH sMH sMsW sMH sW sWH sMsWsH 
5040	167038	2	sWsH sMH sMsW sMH sW sMsWH sMsH 
5040	159024	2	MsH M M MW MsW sMWsH WH 
5040	161032	2	MW W WsH WH MsH MsW sMW 
5040	156973	2	MW W WsH M MH M sMW 
5040	157977	2	MW W WsH M MWH MW sMW 
5040	166614	2	MW W WsH sM sMWH WsH sMsWsH 
5040	166614	2	MW W WsH sMsW sMWH WsH sMsH 
5040	146015	2	MW WH W WsH M MW MWsH 
5040	149023	2	MW WH W WsH M MW sMW 
5040	146023	2	MW WH W WsH MsH MsW MWsH 
5040	149031	2	MW WH W WsH MsH MsW sMW 
5040	145409	2	MW MW WsH M MH M MWsH 
5040	147413	2	MW MW WsH M MH M sMW 
5040	158272	2	MW MWsH WsH sMsW sW sMWsH sMsWsH 
5040	170870	2	MW MsWsH sM sMWsH MsW MW WH 
5040	152281	2	MW sM sMW WsH sMsW sMWsH sMsWsH 
5040	166614	2	MW sMW WsH sM sMWH WsH sMsH 
5040	166614	2	MW sMW WsH sMsW sMWH WsH sMsWsH 
5040	145409	2	MW sMWsH WsH M MH M MWsH 
5040	147413	2	MW sMWsH WsH M MH M sMW 
5040	154977	2	MsW WH W WsH M M MW 
5040	148439	2	MsW WH W WsH M MW MW 
5040	145431	2	MsW WH W WsH M MW sMWsH 
5040	148431	2	MsW WH W WsH MsH MsW MW 
5040	145423	2	MsW WH W WsH MsH MsW sMWsH 
5040	165614	2	MsW sWsH M M MWsH sMsWsH MH 
5040	165614	2	MsW sWsH M MsW MWsH sWsH MH 
5040	179457	2	MsW sWsH MsH MW sM sMW sMWsH 
5040	169870	2	MsW sWsH sMW MWH M MW sMWsH 
5040	185970	2	MsW M MW WsH sH W MsWH 
5040	177025	2	MsW M MWsH WsH sMsW sW sMWsH 
5040	160866	2	MsW MW WH MW MsWsH sM sMWsH 
5040	149025	2	MsW MW WsH M MH M MW 
5040	147021	2	MsW MW WsH M MH M sMWsH 
5040	154975	2	MsW MWH MW WsH M M MW 
5040	178075	2	MsW MsW sW MWsH WsH sM sMWsH 
5040	187067	2	MsW MsW sW MsWsH sMsWH MsW sMWsH 
5040	179407	2	MsW sM sMsWsH MsH MW W sMWsH 
5040	159272	2	MsW sM sMsWsH MsH MW MWsH sMsWsH 
5040	188457	2	MsW sMWH sMsWsH sMW sM sMW sMWsH 
5040	158022	2	MsW sMWsH WH MsH M M MW 
5040	149025	2	MsW sMWsH WsH M MH M MW 
5040	147021	2	MsW sMWsH WsH M MH M sMWsH 
5040	165614	2	MsW sMsWsH M M MWsH sWsH MH 
5040	149281	2	MsW sMsWsH M MsW sW MWsH sMsWsH 
5040	165614	2	MsW sMsWsH M MsW MWsH sMsWsH MH 
5040	145082	2	sMH MsH MsW sMWsH MsW MW WH 
5040	162038	2	sMH sM sMH sW sWH sMsH sWsH 
5040	162038	2	sMH sM sMH sW sWH sMsWsH sH 
5040	162038	2	sMH sM sMH sW sMsWH sMsH sH 
5040	162038	2	sMH sM sMH sW sMsWH sMsWsH sWsH 
5040	175480	2	sMH sMW sMsH W M MW WH 
5040	162038	2	sMH sMsW sMH sW sWH sMsH sH 
5040	162038	2	sMH sMsW sMH sW sWH sMsWsH sWsH 
5040	162038	2	sMH sMsW sMH sW sMsWH sMsH sWsH 
5040	162038	2	sMH sMsW sMH sW sMsWH sMsWsH sH 
5040	161038	2	sMsH sWH sM sMH sW sWH sWsH 
5040	161038	2	sMsH sWH sMsW sMH sW sMsWH sWsH 
5040	161038	2	sMsH sMsWH sM sMH sW sMsWH sWsH 
5040	161038	2	sMsH sMsWH sMsW sMH sW sWH sWsH 
5040	162038	2	sMsW sWH sM sMH sW sWH sMsW 
5040	162038	2	sMsW sWH sM sMH sW sMsWH sM 
5040	162038	2	sMsW sWH sMsW sMH sW sWH sM 
5040	162038	2	sMsW sWH sMsW sMH sW sMsWH sMsW 
5040	162038	2	sMsW sMsWH sM sMH sW sWH sM 
5040	162038	2	sMsW sMsWH sM sMH sW sMsWH sMsW 
5040	162038	2	sMsW sMsWH sMsW sMH sW sWH sMsW 
5040	162038	2	sMsW sMsWH sMsW sMH sW sMsWH sM 
5040	155390	2	sMsWH M M MH W WsH WH 
5040	155390	2	sMsWH M MsW MH W sMWsH WH 
5040	160038	2	sMsWH sM sMH sW sWH sWsH sH 
5040	160038	2	sMsWH sM sMH sW sWH sM sMsW 
5040	160038	2	sMsWH sM sMH sW sMsWH sWsH sMsH 
5040	160038	2	sMsWH sM sMH sW sMsWH sMsW sMsW 
5040	160038	2	sMsWH sMsW sMH sW sWH sWsH sMsH 
5040	160038	2	sMsWH sMsW sMH sW sWH sMsW sMsW 
5040	160038	2	sMsWH sMsW sMH sW sMsWH sWsH sH 
5040	160038	2	sMsWH sMsW sMH sW sMsWH sM sMsW 
5040	156428	2	sMsWsH MH MsW sWsH M M MWsH 
5040	156428	2	sMsWsH MH MsW sMsWsH M MsW MWsH 
5040	156696	2	sMsWsH MW MWsH WsH sMsW sW sMWsH 
5040	146683	2	sMsWsH MW sM sMW WsH sMsW sMWsH 
5040	156696	2	sMsWsH MsW sM sMsWsH MsH MW MWsH 
5040	146683	2	sMsWsH MsW sMsWsH M MsW sW MWsH 

1467.493 million nodes generated
2921484 Compositions found, 108 output
Best score 221923, longest length 5040
